app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -18,7 +18,7 @@ class Admin::AdminSetsController < ApplicationController
     self.member_search_builder_class = Hyrax::AdminAdminSetMemberSearchBuilder
 
     # Used to get a list of admin sets for the index action
-    self.list_search_builder_class = Hyrax::AdminSetSearchBuilder
+    # self.list_search_builder_class = Hyrax::AdminSetSearchBuilder
```

Comment:
Remove commented out code…we have version control. As commented out code, I don't know if it was on accident or not. 

Also, why is this getting removed? If we do in fact want to remove this, make sure to get rid of line 20's comments as well.